### ISSUE TITLE
Command `bin/m build` does not build modules

### ISSUE THREAD

**Issue by trodrigu** [billstclair/elm-port-funnel#2]
Great project!  I am liking the abstractions :)

## The steps to reproduce the bug
1) `git clone https://github.com/billstclair/elm-port-funnel.git`
2) `cd elm-port-funnel/examples`
3) `bin/m build`

## Backtrace
```
- FILE NOT FOUND --------------------------------------------------------------

You want me to compile this file:

    build.elm

I cannot find it though! Is there a typo?

```

**Comment by trodrigu** [comment]
PR #3

**Comment by billstclair** [comment]
Fixed by 1d6c20a
